Form-Controls/styles.css

```diff
@@ -0,0 +1,60 @@
+
+
+/* header */
+.head{
+    display: flex;
+    justify-content: center;
+}
+/* form */
+input:focus {
+    background-color: rgb(0, 255, 225);
+  }
+
+.form{
+    display: flex;
+    justify-content: center;
+    font-size: 20px;
+ }
+
+/* name */
+.name{
+    display: flex;
+    flex-direction: row;
+    font-size: 20px;
+    margin: auto;
+    padding: 6px;
+}
+
+
+/* email */
+
+.email {
+    font-size: 20px;

```

Comment:
Instead of setting the font size on each of these blocks independently, can you think of a general rule, or class, you could apply to everything with the same style?